polymesh_schema.json

```diff
@@ -1581,6 +1596,71 @@
           }
         ],
         "type": "AffirmationCount"
+      },
+      "get_transfer_report": {
```

Comment:
These need to be defined in the `runtime` section of the schema file.  Also I would recommend that we not add new RPC endpoints and only provide them as Runtime APIs.  Substrate is also moving away from defining more RPC endpoints.  They even removed all the RPC endpoints for the `pallet_contracts`, to only provide Runtime API endpoints.